openpathsampling/chaindict.py

```diff
@@ -60,7 +64,7 @@ def __getitem__(self, items):
         return results
 
     def _add_new(self, items, values):
-        self[items] = values
+        pass
```

Comment:
?? purpose of `._add_new` if all it does is `pass`? Is this an abstract that should `raise NotImplementedError`?